app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -0,0 +1,241 @@
+
+## Deployment options
+
+The following instructions assume that you are deploying {{site.base_gateway}} in [classic embedded mode](/enterprise/{{include.kong_version}}/deployment/deployment-options).
+
+If you want to run {{site.base_gateway}} in Hybrid mode, the instructions in this topic will walk you though setting up a Control Plane instance. Afterward, you will need to bring up additional gateway instances for the Data Planes, and perform further configuration steps. See [Hybrid Mode Setup](/enterprise/{{include.kong_version}}/deployment/hybrid-mode-setup) for details.
+
+## Prerequisites
+
+For this installation, you'll need a Docker-enabled system with proper Docker access.
+
+## Install Gateway with a database
+
+1. **Pull the Docker image**
+
+   ```bash
+   docker pull kong/kong-gateway:{{include.kong_versions[13].version}}-alpine
+   ```
+
+   {:.important}
+   > Some [older {{site.base_gateway}} images](https://support.konghq.com/support/s/article/Downloading-older-Kong-versions)
+   are not publicly accessible. If you need a specific patch version and can't
+   find it on [Kong's public Docker Hub page](https://hub.docker.com/r/kong/kong-gateway), contact
+   [Kong Support](https://support.konghq.com/).
+
+   You should now have your {{site.base_gateway}} image locally.
+
+2. **Tag the image**
+
+   ```bash
+   docker tag kong/kong-gateway:{{include.kong_versions[13].version}}-alpine kong
+   ```
+
+3. **Create a Docker network**
+
+   Create a custom network to allow the containers to discover and communicate with each other.
+
+   ```bash
+   docker network create kong-net
+   ```
+
+4. **Start a database**
+
+   Start a PostgreSQL container:
+
+   ```p
+   docker run -d --name kong-database \
+     --network=kong-net \
+     -p 5432:5432 \
+     -e "POSTGRES_USER=kong" \
+     -e "POSTGRES_DB=kong" \
+     -e "POSTGRES_PASSWORD=kong" \
+     postgres:9.6
+   ```
+
+5. **Prepare the Kong database**
+
+   <pre><code>docker run --rm --network=kong-net \
+     -e "KONG_DATABASE=postgres" \
+     -e "KONG_PG_HOST=kong-ee-database" \
+     -e "KONG_PG_PASSWORD=kong" \
+     -e "KONG_PASSWORD=<div contenteditable="true">{PASSWORD}</div>" \
+     kong kong migrations bootstrap</code></pre>
+
+6. **Start Gateway with Kong Manager**
+
+   <pre><code>docker run -d --name kong --network=kong-net \
+     -e "KONG_DATABASE=postgres" \
+     -e "KONG_PG_HOST=kong-database" \
+     -e "KONG_PG_PASSWORD=kong" \
+     -e "KONG_PROXY_ACCESS_LOG=/dev/stdout" \
+     -e "KONG_ADMIN_ACCESS_LOG=/dev/stdout" \
+     -e "KONG_PROXY_ERROR_LOG=/dev/stderr" \
+     -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
+     -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
+     -e "KONG_ADMIN_GUI_URL=http://<div contenteditable="true">{HOSTNAME}</div>:8002" \
+       -p 8000:8000 \
+       -p 8443:8443 \
+       -p 8001:8001 \
+       -p 8444:8444 \
+       -p 8002:8002 \
+       -p 8445:8445 \
+       -p 8003:8003 \
+       -p 8004:8004 \
+       kong</code></pre>
```

Comment:
This will start a Kong Gateway enterprise container; it will fail if you try to run this with an OSS image, as `admin_gui_url` doesn't exist in OSS. 

I wanted to go in an add two codeblocks like for the sections above, but that makes it hard to read. I want to break this into a tabbed codeblock, but I can't do it through github review sueggestions, it's too complicated. Can I go into this branch and make the change directly?